sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/MetricsAdvisorAdministrationClient.cs

```diff
@@ -53,6 +53,37 @@ public MetricsAdvisorAdministrationClient(Uri endpoint, MetricsAdvisorKeyCredent
             _serviceRestClient = new AzureCognitiveServiceMetricsAdvisorRestAPIOpenAPIV2RestClient(_clientDiagnostics, pipeline, endpoint.AbsoluteUri);
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="MetricsAdvisorClient"/> class.
+        /// </summary>
+        /// <param name="endpoint">The endpoint to use for connecting to the Form Recognizer Azure Cognitive Service.</param>
```

Comment:
to the Metrics Advisor ... service

Reply:
I was pretty sure my copy-pasting wouldn't be exposed. I failed this time.

Reply:
My next PR will update the README with AAD info, so I'll include these doc fixes there.